dashboard/models.py

```diff
@@ -621,6 +621,8 @@ class Timepoint(db.Model):
     __tablename__ = 'timepoints'
 
     name = db.Column('name', db.String(64), primary_key=True)
+    bids_name = db.Column('bids_name', db.Text)
```

Comment:
is `bids_name` capturing the participant ID or the basename of the BIDS-converted file?

Reply:
In the timepoint class it's capture the participant ID (e.g. bids_name = 'CMH410008' and bids_session =  '02' for STOPPD_CMH_410008_02) and for scan it's capturing the entire BIDS file name.